include/swift/Sema/CSFix.h

```diff
@@ -1335,8 +1335,8 @@ class UseWrappedValue final : public ConstraintFix {
         PropertyWrapper(propertyWrapper), Base(base), Wrapper(wrapper) {}
 
   bool usingProjection() const {
-    auto nameStr = PropertyWrapper->getName().str();
-    return !nameStr.startswith("_");
+    auto name = PropertyWrapper->getName();
+    return !name.hasUnderscoredNaming();
```

Comment:
We could shorten this even further now

```suggestion
    return !PropertyWrapper->getName().hasUnderscoredNaming();
```

Reply:
@ahoppen Thanks! I like this better but hadn't thought of it. With suggestions is there a preference wether I do it locally so I can squash or is it okay to apply it here?

Reply:
Squashing would be appreciated.

Reply:
Alright done. Thank you.